### PR TITLE
Move elements inside main tag. Fixes #280

### LINKED ISSUE #280: Content must be inside of landmark elements
**Describe the bug**
AXE is flagging the following content as being outside of landmark elements:

- Beta banner
- Breadcrumbs
- Feedback loop banner

**Expected behaviour**
Content should be within the `<header>`, `<main>` or `<footer>` tags

### PR DESCRIPTION
**Pull request by gldgrnt**
Moved beta banner, breadcrumbs and feedback loop inside the main landmark tag. Fixes #280

### CODE REVIEW

**Comment by gldgrnt** [comment]
The changes I've made are visible here: [https://deploy-preview-296--wmnetwork.netlify.app/patterns/header-and-footer/](https://deploy-preview-296--wmnetwork.netlify.app/patterns/header-and-footer/).

It does, however, add an extra `<div>` below the `<header>` and `<footer>` components

**Review by catiarodriguescosta — APPROVED** [review]
Seems all good for me!

File: src/wmnds/patterns/header/_header.njk [hunk 1]
```diff
@@ -16,67 +18,93 @@
...
+        link: 'https://github.com/wmcadigital/wmn-design-system/issues',
+        linkTitle: 'WMN Design System Github'
+      }) | trim | indent(4)
+    -}}
+  </div>
+  {%- endif -%}
+  
+  {%- if params.breadcrumbs and params.breadcrumbs.section != 'Home' %}
+  <!-- Breadcrumbs -->
+  <div class="wmnds-container">
```
Comment (daylesalmon) [inline src/wmnds/patterns/header/_header.njk:99]: I think you could wrap these two components with just one "wmnds-container" wrapper.
You could then do an if statement around the master wmnds-container to say "if params.banner OR params.breadcrumbs" and keep your if statements inside to render breadcrumb/banner if available.

File: src/www/pages/patterns/header-and-footer/index.njk [hunk 2]
```diff
@@ -112,6 +112,21 @@
...
+    wmndsHeader({
+      navItems: navItems,
+      banner: true,
+      breadcrumbs: {
+        section: section,
+        pageTitle: pageTitle
+      }
+    })
+  ])
+}}
```
Comment (daylesalmon) [inline src/www/pages/patterns/header-and-footer/index.njk:129]: Can you move this example to the top of the page as this is our preferred header that we want people to use.

**Review by daylesalmon — CHANGES_REQUESTED** [review]
Looking good, just a few changes.

**Comment by gldgrnt** [comment]
I've replaced the initial example with the full example. We might need to request some copy for the initial example as it now contains the phase banner and the breadcrumbs.

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**